aries-site/src/examples/components/avatar/AvatarSizePreviewExample.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { Avatar, Box, Text } from 'grommet';
+
+export const AvatarSizePreviewExample = () => (
+  <Box align="center" height="small" direction="row" gap="xlarge">
```

Comment:
```suggestion
  <Box align="center"direction="row" gap="xlarge">
```
Feels strange to restrict the height. If we want a small height, we can apply `height={{ min: 'small' }}` to the Example container itself.